js/layout-javascript/agenda-code.js

```diff
@@ -2403,7 +2411,37 @@ DynamicList.prototype.addDetailViewData = function(entry) {
       content = entry.originalData[dynamicDataObj.column];
     }
 
-    content = _this.Utils.String.toFormattedString(content);
```

Comment:
What happened to what `_this.Utils.String.toFormattedString()` used to do? I assume we still need the ability to format `content` appropriately into string. I believe we introduced it to make sure arrays are correctly joined with `", "`.

Reply:
@tonytlwu I'm not sure why this line was removed. But I believe it shouldn't. I'll move it back.